cmd/virt-handler/virt-handler.go

```diff
@@ -102,42 +102,36 @@ func main() {
 		panic(err)
 	}
 
-	l, err := labels.Parse(fmt.Sprintf(v1.NodeNameLabel+" in (%s)", *host))
-	if err != nil {
-		panic(err)
-	}
+	// Create all the informers used by virt-handler here
+	informerFactory := kubeinformers.NewKubeInformerFactory(restClient, clientSet)
+	vmOnHostInformer := informerFactory.VmOnHost(*host)
+	domainSharedInformer := informerFactory.CustomInformer("domainInformer", func() cache.SharedIndexInformer {
```

Comment:
The domain informer is specific to virt-handler, that's why I created the "CustomInformer" function.  It lets the InformerFactory manage shared informers that should not be exposed cluster wide.